### PR TITLE
MAINT-28293: Add limitation for organization and job title fields.

### PR DESCRIPTION
**Pull request by mdiha**
I added fields checks in order that Job title and Organization fields must not start or end with a white space.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue [hunk 1]
```diff
@@ -70,6 +70,7 @@ export default {
   methods: {
     save() {
       this.error = null;
+      const regex = /^[^\s]+(\s+[^\s]+)*$/;
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue]: Use String.trim() and check that the text is bigger than minimum field length

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue (hunk 1 above)
Comment (mdiha) [reply webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue]: So no need to display a warning message ?

Comment (ahamdi) [reply webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue]: For me yes, simply we get rid of all spaces and make the text get checked with the default rules.

Comment (ahamdi) [reply webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue]: @mzorai WDYT ?

Comment (mzorai) [reply webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue]: it's OK for me to trim the seized value at the begining and the end from spaces

Comment (mdiha) [reply webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue:73]: If field contains more then a white space at the beginning or the end we end up in the same situation. WDYT ?

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**